guide/interactions/slash-commands.md

```diff
@@ -196,27 +196,27 @@ client.on('interactionCreate', interaction => {
 });
 ```
 
-However, not every interaction is a slash command (e.g. `MessageComponent`s). Make sure to only receive slash commands by making use of the `CommandInteraction#isCommand()` method:
+However, not every interaction is a slash command (e.g. `MessageComponent`s). Make sure to only receive slash commands by making use of the `Interaction#isChatInputCommand()` method:
```

Comment:
```suggestion
However, not every interaction is a slash command (e.g. `MessageComponent`s). Make sure to only receive slash commands by making use of the `BaseInteraction#isChatInputCommand()` method:
```

`Interaction` is just a union type now. `BaseInteraction` in the class. Same applies to `Channel`. So if  `Channel` class is mentioned anywhere replace it with `BaseChannel`